apps/server/src/form-instances/form-instances.service.ts

```diff
@@ -316,9 +316,12 @@ export class FormInstancesService {
     return updatedFormInstance;
   }
 
-  async markFormInstanceAsCompleted(formInstanceId: string) {
+  async markFormInstanceAsCompleted(
+    employeeId: string,
+    formInstanceId: string,
+  ) {
     const formInstance = await this.prisma.formInstance.findUnique({
-      where: { id: formInstanceId },
+      where: { id: formInstanceId, originatorId: employeeId },
```

Comment:
we should probably make some follow up tickets on this, since i think admins who belong to the same department as the originator should be able to mark the form instance as completed as well

Reply:
originators or admins from the same department can mark form instances as completed